packages/bun-types/bun.d.ts

```diff
@@ -6241,7 +6297,7 @@ declare module "bun" {
     /**
      * The TLS configuration object with which to create the server
      */
-    tls?: TLSOptions;
+    tls?: TLSOptions | boolean;
```

Comment:
Add to JSDoc why `boolean` is supported here/highlight the difference between the object or just a boolean